scripts/migrations/6

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# Migration 6 --> 7
+
+# Delete cert-manager crds labels and annotations
+
+for resource in certificaterequests.cert-manager.io certificates.cert-manager.io challenges.acme.cert-manager.io clusterissuers.cert-manager.io issuers.cert-manager.io orders.acme.cert-manager.io
+do
+    kubectl annotate crd $resource meta.helm.sh/release-name-
+    kubectl annotate crd $resource meta.helm.sh/release-namespace-
+
+    kubectl label crd $resource app-
+    kubectl label crd $resource app.kubernetes.io/instance-
+    kubectl label crd $resource app.kubernetes.io/managed-by-
+    kubectl label crd $resource app.kubernetes.io/name-
+    kubectl label crd $resource app.kubernetes.io/version-
+    kubectl label crd $resource helm.sh/chart-
+    kubectl label crd $resource helm.toolkit.fluxcd.io/name-
+    kubectl label crd $resource helm.toolkit.fluxcd.io/namespace-
+done
```

Comment:
This is not going to work, because cert-manager-crds will not adopt resources without annotations

```suggestion
# Fix cert-manager crds
certmanager_crds=$(kubectl get crd -o name | grep '\.cert-manager\.io$')
if [ -n "$certmanager_crds" ]; then
  kubectl annotate $kubeovn_crds meta.helm.sh/release-namespace=cozy-cert-manager meta.helm.sh/release-name=cert-manager-crds
  kubectl label $kubeovn_crds app.kubernetes.io/managed-by=Helm
fi
```